configs.d/input/debug.conf

```diff
@@ -0,0 +1,5 @@
+<source>
+  @type debug_agent
```

Comment:
BTW, @richm did you find any information about how this `debug_agent` can be used? I do not think this is exposed via HTTP/REST in fluentd.